### DISCUSSION TITLE
My request accepted but didn't get benefits yet

### DISCUSSION THREAD

**Discussion by princeiiti** [community/community#135142] [GitHub Education]
### Hi everyone,

Hii i applied for student benefits 6 days ago, but it still shows - Your request was approved. [Your benefits will be available soon.]
What should i do?

**Comment by davevad93** [comment]
Hi @princeiiti ,

As stated in the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352), If you haven't received your benefits within 5 days of your approval status you should raise a discussion here as you have done to inform the Education team in order for them to assist you. However, the waiting time may vary depending on different factors, such as the volume of requests. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @princeiiti was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279

And also:

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
